client-app/shared/payment/components/bank-card-form.vue

```diff
@@ -117,38 +123,50 @@ const labels = computed(() => {
   };
 });
 
-const monthYupSchema = string()
+const monthYupSchema = yup
+  .string()
   .required()
   .length(2)
   .matches(/^(0?[1-9]|1[0-2])$/, t("shared.payment.authorize_net.errors.month"))
   .label(labels.value.monthLabel);
 
 const validationSchema = toTypedSchema(
-  object({
-    number: string().required().min(12).max(19).label(labels.value.number),
-    cardholderName: string().required().max(64).label(labels.value.cardholderName),
+  yup.object({
+    number: yup.string().required().min(12).max(19).label(labels.value.number),
+    cardholderName: yup.string().required().max(64).label(labels.value.cardholderName),
     month: monthYupSchema,
-    year: string().when("month", ([month], schema) => {
+    year: yup.string().when("month", ([month], schema) => {
       return monthYupSchema.isValidSync(month) ? schema.length(2).label(labels.value.yearLabel) : schema;
     }),
-    securityCode: string().required().min(3).max(4).label(labels.value.securityCode),
+    securityCode: yup.string().required().min(3).max(4).label(labels.value.securityCode),
   }),
 );
 
-const { values, meta, errors: formErrors } = useForm<BankCardType>({ validationSchema, initialValues });
+const {
+  values,
+  meta,
+  errors: formErrors,
+  defineField,
+} = useForm({
+  validationSchema,
+  initialValues,
+});
 
-const { value: number } = useField<string>("number", undefined, { syncVModel: false });
-const { value: cardholderName } = useField<string>("cardholderName", undefined, { syncVModel: false });
-const { value: month } = useField<string>("month", undefined, { syncVModel: false });
-const { value: year } = useField<string>("year", undefined, { syncVModel: false });
-const { value: securityCode } = useField<string>("securityCode", undefined, { syncVModel: false });
+const [number] = defineField("number");
```

Comment:
const FIELD_IDS = {
 ...
}

Reply:
Same. And vee-validate provide typing for that, so if you'll try to use wrong name linter & typescript compiler will say you